packages/babel-cli/test/index.js

```diff
@@ -14,12 +14,10 @@ const fileFilter = function(x) {
   return x !== ".DS_Store";
 };
 
-const presetLocs = [
-  path.join(__dirname, "../../babel-preset-es2015"),
```

Comment:
Was going to switch to env but I don't think we really need to test for it if it's just the cli?